Setup.ps1

```diff
@@ -32,18 +32,19 @@ Function Script:Set-EnvironmentVariable {
      if($Scope -eq 'Machine') {
          $scopeArgs = '/M'
      }
-     setx.exe $Name $Value $scopeArgs | Out-Null
+
+     [System.Environment]::SetEnvironmentVariable($Name,$Value,[System.EnvironmentVariableTarget]::Machine)
      Set-Item -Path Env:$Name -Value $Value
 }
 
 
 $PSToolboxPath=Join-Path $PSScriptRoot Modules
-if($env:PSModulePath -notlike "*$PSToolboxPath*") {
-    Script:Set-EnvironmentVariable -Name 'PSModulePath' -Value "$PSToolboxPath;$env:PSModulePath" 
+if($env:PSModulePath -notlike "$PSToolboxPath*") {
+    $UpdatedPSModulePath = "$PSToolboxPath;$(($env:PSModulePath -split ';' | Where-Object{ $_ -ne 'C:\Git\PSToolbox\Modules' })  -join ';')"

```

Comment:
Will this work on my machine? `C:\Git\PSToolbox\Modules` isn't the path to my source tree. Should it be `$pwd\Modules`?